lesson22/app.js

```diff
@@ -1,225 +1,234 @@
-const url = "https://jsondata.okiba.me/v1/json/tN7Ay210317141032";
 const parent = document.getElementById("js-parent");
-const table = document.createElement("table");
 
-table.id = "table";
-table.classList.add("table");
+const createAttributedElements = (tag, attrObj, str = "") => {
```

Comment:
[valuesByKeys](https://softwareas.com/naming-conventions-for-key-value-maps/)にならって、valuesByAttributes

```suggestion
const createAttributedElements = (tag, valuesByAttributes, str = "") => {
```